app/scss/3-views/user/profile.scss

```diff
@@ -134,4 +134,8 @@
     margin: 5px 0;
   }
 }
+.cui-star-password{
```

Comment:
Just a note here in that this _may_ want to be maintained in the style guide as a common approach to handling masked inputs.  You may have already considered that, but don't be afraid to continue boosting the features in styleguide styles alongside the B2X implementation.